theme/forms.py

```diff
@@ -114,19 +110,20 @@ def clean_honeypot(self):
             raise forms.ValidationError(self.fields["honeypot"].label)
         return value
 
+
 # added by Hong Yi for customizing THreadedCommentForm
 class ThreadedCommentForm(CommentForm, Html5Mixin):
 
-    #name = forms.CharField(label=_("Name"), help_text=_("required"),
+    # name = forms.CharField(label=_("Name"), help_text=_("required"),
```

Comment:
@pkdash These commented code can be removed. Will remove them tomorrow. Let me know if you have other comments on this PR so that I can address them together tomorrow.